winterfell/controllers/authController.js

```diff
@@ -1,18 +1,17 @@
 var _ = require('lodash');
-var passport = require('passport');
 var constants = require('./../utils/constants');
 var http = require('http-status-codes');
 var httpErrors = require('./../utils/httpErrors');
+var Log = require('./../utils/logHelper');
+var passport = require('passport');
 var User = require('./../models/user');
 var UserValues = require('./../models/userValues');
 var UserService = require('./../services/userService');
 
 module.exports = function (app) {
 
   // Endpoint for routing sign-up
-  app.get('/signup', function(req, res) {
-    console.log('[signup] request received for ' + JSON.stringify(req.body));
-    console.log('[signup] session for ' + JSON.stringify(req.session));
+  app.get('/signup', [Log.api], function(req, res) {
```

Comment:
I was just using a couple of different ways to log requests (including `morgan`, `bunyan-express`, `bunyan-request`, bunyan serializers) so we don't need to include this middleware everywhere. The problem with all of these is that with express, EVERYTHING gets logged. When you refresh the home page, logs for GET requests for assets, fonts, pictures, etc. all get logged - which becomes extremely noisy. 

I don't think those logs should get logged, only our APIs like `GET /user`. Which is why I did it the way I had it before (log only for every endpoint). I'm not sure if there's an easy way to filter out express only API requests.